pom.xml

```diff
@@ -237,12 +237,12 @@
       </dependency>
       <dependency>
         <groupId>io.opentelemetry</groupId>
-        <artifactId>opentelemetry-sdk</artifactId>
+        <artifactId>opentelemetry-sdk-trace</artifactId>
         <version>${opentelemetry.version}</version>
       </dependency>
       <dependency>
         <groupId>io.opentelemetry</groupId>
-        <artifactId>opentelemetry-exporters-inmemory</artifactId>
+        <artifactId>opentelemetry-sdk-testing</artifactId>
```

Comment:
Should this dependency be in the test scope.

Reply:
Pretty sure I had test scope for the module importing it, but can add here for safety